package.json

```diff
@@ -40,7 +40,8 @@
 		"test:update": "wp-scripts test-unit-js --updateSnapshot --config tests/js/jest.config.json",
 		"test:watch": "npm run test -- --watch",
 		"size-check": "bundlewatch",
-		"changelog": "node ./bin/changelog"
+		"changelog": "node ./bin/changelog",
+		"changelog:zenhub": "node ./bin/changelog --changelogSrcType:'MILESTONE'"
```

Comment:
Can’t we skip this option and directly pass it via env?

Reply:
We can, but the purpose of npm scripts is to save on typing and it's easier to remember.  So (apart from fixing the typo), I'm inclined to leave this.